iothub/device/tests/IotHubDeviceClientTests.cs

```diff
@@ -614,16 +560,13 @@ await deviceClient
                     {
                         isMethodHandlerCalled = true;
                         responseReceivedAsExpected = payload.TryGetPayload(out response);
-                        int? connectionTimeout = payload.ConnectionTimeoutInSeconds;
-                        int? responseTimeout = payload.ResponseTimeoutInSeconds;
                         return Task.FromResult(_directMethodResponseWithPayload);
                     })
                 .ConfigureAwait(false);
 
             const string payload = "test";
-            var directMethodRequest = new DirectMethodRequest("TestMethodName")
+            var directMethodRequest = new DirectMethodRequest("TestMethodName", DefaultPayloadConvention.Instance.GetObjectBytes(payload))
```

Comment:
Instead of referencing newtonsoft directly, using the default convention to get the bytes